src/main/java/com/faforever/client/api/FafApiAccessor.java

```diff
@@ -157,6 +157,11 @@ public Mono<MeResult> getMe() {
         .doOnNext(object -> log.trace("Retrieved {} from /me with type MeResult", object));
   }
 
+  public Mono<IceSession> getIceSession(int gameId) {
+    return retrieveMonoWithErrorHandling(IceSession.class, apiWebClient.get().uri("/ice/session/game/" + gameId))
+        .doOnNext(object -> log.trace("Retrieved {} from /ice/session/game/{} with type MeResult", object, gameId));
```

Comment:
We can't use the retrieveMonoWithErrorHandling since that expects JsonApiDocument errors

Reply:
What can we use instead?

Reply:
RetrieveWithErrorHandling is just a wrapper around common error handling so we would just make the call with the required custom error handling